src/sonic-host-services/scripts/caclmgrd

```diff
@@ -319,6 +327,77 @@ class ControlPlaneAclManager(daemon_base.DaemonBase):
         else:
             return False
 
+    def setup_dhcp_chain(self, namespace):
+        all_chains = self.get_chain_list(self.iptables_cmd_ns_prefix[namespace], [""])
+        dhcp_chain_exist = 1 if "DHCP" in all_chains else 0
```

Comment:
what about using boolean?
`dhcp_chain_exist = "DHCP" in all_chains`

Reply:
updated